src/matrix/common.js

```diff
@@ -22,16 +22,16 @@ export function makeTxnId() {
 }
 
 export function isTxnId(txnId) {
-	return txnId.startsWith("t") && txnId.length === 15;
+    return txnId.startsWith("t") && txnId.length === 15;
 }
 
 export function tests() {
-	return {
```

Comment:
Please don't change the indenting when not making changes.

Reply:
Do I have to cancel those indenting changes or it should stay like that?